hawc/apps/epi/api.py

```diff
@@ -16,6 +17,7 @@ class EpiAssessmentViewset(
     parent_model = Assessment
     model = models.Outcome
     permission_classes = (AssessmentLevelPermissions,)
+    serializer_class = ser.Serializer
```

Comment:
serializer_class was required for OpenAPI, so I used the base Serializer class for viewsets that didn't have their serializer_class set. I considered using serializers that best matches the models being used, but I was concerned that would make the code harder to read since people may assume the serializer was actually being used. The tradeoff is a nondescript "response content" portion of the OpenAPI spec.

Reply:
This looks great. Can we subclass the serializer class and put in the common package and call it something like `UnusedSerializer` with a docstring that describes what you wrote above? Then it'd be easy to find all the ones to fix.

Reply:
Good idea, done!